### PR TITLE
Change lexical scope serial

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: src/js/port_handler.js [hunk 1]
```diff
@@ -90,9 +90,6 @@ PortHandler.check_usb_devices = function (callback) {
             callback(self.dfu_available);
         }
         if (!$('option:selected', self.portPickerElement).data().isDFU) {
-            if (!GUI.connected_to) {
```
Comment (mikeller) [inline src/js/port_handler.js:93]: Why is this removed?

File: src/js/serial.js [hunk 2]
```diff
@@ -69,13 +69,13 @@ const serial = {
                             self.error = info.error;
                             setTimeout(function() {
                                 chrome.serial.setPaused(info.connectionId, false, function() {
-                                    self.getInfo(function (info) {
-                                        if (info) {
-                                            if (info.paused) {
+                                    self.getInfo(function (getInfo) {
```
Comment (mikeller) [inline src/js/serial.js:72]: This rename seems to make the code less intuitive and harder to read (`getInfo` would be a getter for `info`, and not for `getInfo`).

File: src/js/serial.js [hunk 3]
```diff
@@ -299,14 +301,14 @@ const serial = {
...
-                        var counter = 0;
+                        let counter = 0;
 
                         while (self.outputBuffer.length > 100) {
                             self.outputBuffer.pop();
                             counter++;
                         }
 
-                        console.log(`${self.connectionType}: send buffer overflowing, dropped: ${counter} ${entries}`);
+                        console.log(`${self.connectionType}: send buffer overflowing, dropped: ${counter}`);
```
Comment (mikeller) [inline src/js/serial.js:311]: Why is this removed?

File: src/js/serial_backend.js [hunk 4]
```diff
@@ -676,33 +672,35 @@ function update_live_status() {
...
-               else
-                       $(".armedicon").removeClass('active');
-       }
-       if (FC.AUX_CONFIG[i] == 'FAILSAFE') {
-               if (bit_check(FC.CONFIG.mode, i))
-                       $(".failsafeicon").addClass('active');
-               else
-                       $(".failsafeicon").removeClass('active');
-       }
+    for (const index in FC.AUX_CONFIG) {
```
Comment (mikeller) [inline src/js/serial_backend.js:677]: `for ( of )` or leave as is.

**Review by mikeller — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Thanks. Fixed as requested per review

File: src/js/serial.js (hunk 2 above)
Comment (haslinghuis) [reply src/js/serial.js]: info is already used in upper scope. Like to have an exclusive object for the function. Changed it to `_info`

**Review by mikeller — APPROVED** [review]

**Pull request merged**